include/com/centreon/engine/engine_impl.hh

```diff
@@ -64,6 +64,7 @@ class engine_impl final : public Engine::Service {
   grpc::Status GetHostDependenciesCount(grpc::ServerContext* context,
                                         const ::google::protobuf::Empty*,
                                         GenericValue*) override;
+  grpc::Status GetHost(grpc::ServerContext* context, const HostIdentifier*, EngineHost*) override;
```

Comment:
I suspect you did not set a name to the HostIdentifier* to copy what was done before.

In fact, as you can see, ::google::protobuf::Empty* represents an empty object, so it will be never used (because empty)., so to avoid a warning telling the variable is not used or something like that, we prefered not setting a name.

In your case, it is different, when you implement the method you need a name, and you can set it there.